app/src/main.c

```diff
@@ -235,6 +240,9 @@ static SDL_bool parse_args(struct args *args, int argc, char *argv[]) {
                     return SDL_FALSE;
                 }
                 break;
+            case 'o':
+                args->outfilename = optarg;
```

Comment:
To make consistent, use underscore as delimiter.

Reply:
OK